.github/workflows/release.yml

```diff
@@ -56,8 +101,63 @@ jobs:
           git commit -a -m "Update changelog"
           git push "https://${{ env.CI_COMMIT_AUTHOR }}:${{ secrets.TF_DEVEX_COMMIT_GITHUB_TOKEN }}@github.com/${{ github.repository }}.git"
 
+  cdktfDocs:
+    needs:
+      - releasedVersion
+      - buildBinary
+      - changelog # So the commit is pushed for the changelog
+    runs-on: ubuntu-latest
+    container:
+      image: docker.mirror.hashicorp.services/hashicorp/jsii-terraform
+    env:
+      CHECKPOINT_DISABLE: "1"
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - run: git config --global user.email "bcroft@hashicorp.com"
+      - run: git config --global user.name "Brandon Croft"
+      - name: Get yarn cache directory path
+        id: global-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+      - uses: actions/cache@v3.0.11
+        id: global-cache
+        with:
+          path: ${{ steps.global-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-integration-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
+
+      - name: Install cdktf-registry-docs
+        run: npm install -g cdktf-registry-docs@1.10.1
```

Comment:
This feels like it should be moved to a `package.json` file, similar to how the Digital team manages website packages. This will enable the dependency to be managed by Dependabot for updates, etc.

Reply:
I'm not sure about this yet. Once we got more repositories onboarded to the workflow I'd expect certain patterns to emerge. Once we are there we might move all of this into a reusable workflow where inlining the used version of the tool would make the most sense.